### PR TITLE
[DEVSU-2160] change user sig text

### PR DESCRIPTION
**Pull request by kttkjl**

### CODE REVIEW

File: app/hooks/useConfirmDialog.tsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import ReactDOM from 'react-dom';
...
 
 const textDict = {
-  probe: 'Do you wish to remove Ready and Reviewer signatures?',
-  pharmacogenomic: 'Do you wish to remove Ready and Reviewer signatures?',
-  genomic: 'Do you wish to remove Author and Reviewer signatures?',
-  rapid: 'Do you wish to remove Author and Reviewer signatures?',
+  probe: 'Making this change will cause signatures to be removed. do you want to proceed?',
+  pharmacogenomic: 'Making this change will cause signatures to be removed. do you want to proceed?',
+  genomic: 'Making this change will cause the reviewer and analyst signatures to be removed. do you want to proceed?',
+  rapid: 'Making this change will cause the reviewer and analyst signatures to be removed. do you want to proceed?',
```
Comment (elewis2) [inline app/hooks/useConfirmDialog.tsx:12]: capitalize 'Do' for each

**Review by elewis2 — CHANGES_REQUESTED** [review]
When the signatures are removed, the report status should drop back down to 'ready' (currently it is still 'reviewed' even after hard refresh).

File: app/hooks/useConfirmDialog.tsx (hunk 1 above)
Comment (kttkjl) [reply app/hooks/useConfirmDialog.tsx:12]: 🤦  oh dear copy and paste

Comment (kttkjl) [reply app/hooks/useConfirmDialog.tsx]: [ea69abc](https://github.com/bcgsc/pori_ipr_client/pull/402/commits/ea69abc2c2f90546ead1199e7da4e1a321ef5f95)

**Review by elewis2 — APPROVED** [review]

**Pull request merged**